docs/public/_redirects

```diff
@@ -0,0 +1,3 @@
+# Proxies
+# material-ui-x is configured to be hosted at the root.
+/static/* https://material-ui.netlify.app/static/:splat 200
```

Comment:
should fix https://master--material-ui-x.netlify.app/components/data-grid/getting-started/#github

<img width="180" alt="Capture d’écran 2020-09-11 à 00 18 40" src="https://user-images.githubusercontent.com/3165635/92815276-5e552780-f3c4-11ea-8c4a-2d48fbdecb59.png">